target/linux/qualcommax/ipq807x/base-files/lib/upgrade/platform.sh

```diff
@@ -51,9 +60,6 @@ platform_do_upgrade() {
 		nand_do_restore_config || nand_do_upgrade_failed
 		buffalo_upgrade_optvol
 		;;
-	dynalink,dl-wrx36)
```

Comment:
I get what you are trying to do here by cleaning all of these that use the same, but please separate commit.